ambry-router/src/main/java/com/github/ambry/router/QuotaAwareOperationController.java

```diff
@@ -138,8 +140,11 @@ private void addToRequestQueue(List<RequestInfo> requestInfos) {
    */
   private void drainRequestQueue(List<RequestInfo> requestsToSend) {
     for (QuotaMethod quotaMethod : QuotaMethod.values()) {
-      pollQuotaCompliantRequests(requestsToSend, getRequestQueue(quotaMethod));
-      pollQuotaExceedAllowedRequestsIfAny(requestsToSend, getRequestQueue(quotaMethod));
+      Map<QuotaResource, LinkedList<RequestInfo>> queue = getRequestQueue(quotaMethod);
+      pollQuotaCompliantRequests(requestsToSend, queue);
+      outOfQuotaRequestsInQueue = queue.size();
```

Comment:
Yes.. out of quota requests are those requests that exceed the quota. Some of them will still be served because we allow accounts to exceed their quota if frontend has enough capacity left. But some of the out of requests might be delayed if frontend capacity usage is at 80% or more.